torch/distributed/launch.py

```diff
@@ -1,7 +1,6 @@
 r"""
 `torch.distributed.launch` is a module that spawns up multiple distributed
 training processes on each of the training nodes.
-
```

Comment:
why were these spaces removed? looks like they were actually there to split paragraphs